.ci/setup-drupal-repo.sh

```diff
@@ -21,13 +21,19 @@ git checkout -b $MULTIDEV_NAME
 
 composer -- config repositories.secrets vcs https://github.com/pantheon-systems/pantheon_secrets.git
 
-# dev-1.0.x does not match anything, should be 1.0.x-dev as per https://getcomposer.org/doc/articles/aliases.md#branch-alias.
-export BRANCH_PART="dev-${GIT_BRANCH}"
-if [ $GIT_BRANCH = "1.0.x" ]; then
-  export BRANCH_PART="1.0.x-dev"
+export SECRETS_VERSION=$GIT_BRANCH
+
+if [ $GIT_REF_TYPE = "branch" ]; then
+  # dev-1.0.x does not match anything, should be 1.0.x-dev as per https://getcomposer.org/doc/articles/aliases.md#branch-alias.
+  export BRANCH_PART="dev-${GIT_BRANCH}"
+  if [ $GIT_BRANCH = "1.0.x" ]; then
+    export BRANCH_PART="1.0.x-dev"
```

Comment:
Maybe replace `.x` with `.x-dev` so this doesn't need to be changed when the branch version changes?